account_payment_partner/wizard/payment_order_create.py

```diff
@@ -28,10 +28,10 @@ class PaymentOrderCreate(models.TransientModel):
 
     @api.model
     def extend_payment_order_domain(self, payment_order, domain):
-        res = super(PaymentOrderCreate, self).extend_payment_order_domain(
+        super(PaymentOrderCreate, self).extend_payment_order_domain(
             payment_order, domain)
         domain += ['|', '|',
                    ('invoice', '=', False),
                    ('invoice.payment_mode_id', '=', False),
                    ('invoice.payment_mode_id', '=', payment_order.mode.id)]
-        return res
+        return True
```

Comment:
in general it's good to propagate the return from super (unless you really want to change it). Here super will return True too but eventually it could be changed. Is there any reason to hardcode return True in this override? Not against it, but just asking why this choice. (approve the file removal otherwise).